### PR TITLE
Gnome Ethnicity

### PR DESCRIPTION
**Pull request by ercarp**
## Changelog:
- Made an ethnicity for gnomes so they don't have to use dwarven ethnicity in-game anymore.
- Made it the active ethnicity for gnomish cultures.

# How to test:
There are two ways to test this:
1. Open up Ruler Designer and generate male and female gnomes to see what they look like.
2. Alternatively, load up into a game and look for gnomes in-game using the character search panel.

**Note**: Nose, eye, and ear sizes are maxed out. I know they're bigger in WoW, but this is the best we can get unless the 3D wizards figure out a way to make them even bigger.

### CODE REVIEW

File: common/ethnicities/wc_ethnicities_gnome.txt [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+	#Upper Lip
+	gene_mouth_upper_lip_size = {
+		1 = { name = mouth_upper_lip_size_neg    range = { 0.2 0.6 }    }
+	}
+	gene_bs_mouth_upper_lip_def = {
+		1 = { name = mouth_upper_lip_def_pos    range = { 0 1 }    }
+	}
+	gene_bs_mouth_upper_lip_full = {
+		1 = { name = mouth_upper_lip_full_pos    range = { 0 1 }    }
+		1 = { name = mouth_upper_lip_full_neg    range = { 0 0.65 }  z  }
```
Comment (zumbak04) [inline common/ethnicities/wc_ethnicities_gnome.txt:135]:
```suggestion
		1 = { name = mouth_upper_lip_full_neg    range = { 0 0.65 }    }
```

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/ethnicities/wc_ethnicities_gnome.txt (hunk 1 above)
Comment (ercarp) [reply common/ethnicities/wc_ethnicities_gnome.txt]: Fixed in a0bfdc5d1cab597195c6649286e610261b73135e!

File: common/ethnicities/wc_ethnicities_gnome.txt [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+		1 = { name = mena_hairstyles 		range = { 0 1 } } # very good for male gnomes, not so much for females
+		1 = { name = byzantine_hairstyles 		range = { 0.82 0.82 } } #male_hair_byzantine_02
+	}
+	beards = {
+		1 = { name = western_beards 		range = { 0.49 0.92 } }
+		1 = { name = mena_beards 		range = { 0.40 0.74 } }
+		1 = { name = steppe_beards 		range = { 0 1 } }
+		1 = { name = sub_saharan_beards 		range = { 0.92 0.92 } } #male_beard_sub_saharan_03
+		1 = { name = indian_beards 		range = { 0 1 } }
+	}
```
Comment (zumbak04) [inline common/ethnicities/wc_ethnicities_gnome.txt:370]: Create beard and hair genes here.

File: common/ethnicities/wc_ethnicities_gnome.txt [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+        1 = { name = head_profile_neg    range = { 0.25 0.5 }    }
+    }
+	
+	#Head top
+	gene_head_top_height = {
+        1 = { name = head_top_height_neg    range = { 0 0.33  }    } # REMEMBER TO ADJUST RANGE!
+	}
+	gene_head_top_width = {
+        1 = { name = head_top_width_neg    range = { 1 1 }    } # REMEMBER TO ADJUST RANGE!
+	}
```
Comment (zumbak04) [inline common/ethnicities/wc_ethnicities_gnome.txt:294]: **Remember remember...**

File: common/ethnicities/wc_ethnicities_gnome.txt (hunk 2 above)
Comment (ercarp) [reply common/ethnicities/wc_ethnicities_gnome.txt]: > 
> 
> Create beard and hair genes here.

Added in 6b389930b94a7a178d0bb1f07d2d6dcc0b76daab.

File: common/ethnicities/wc_ethnicities_gnome.txt (hunk 3 above)
Comment (ercarp) [reply common/ethnicities/wc_ethnicities_gnome.txt]: Adjusted these in 6b389930b94a7a178d0bb1f07d2d6dcc0b76daab.

**Comment by ercarp** [comment]
Changed base branch directly to `master` as the gnomes don't need the troll genes for anything.

**Comment by zumbak04** [comment]
> Changed base branch directly to `master` as the gnomes don't need the troll genes for anything.

There's another reason, the gnome branch has the troll branch in it. So, I'll rebase it on the trolls again. Once the trolls are merged, the gnomes will be rebased on the master automatically.

**Review by zumbak04 — APPROVED** [review]
They're horrifying. I love them.

**Review by TheDrPibb1999 — APPROVED** [review]
Seriously these random gens made me laugh extremely hard, these are perfect for gnomes. I can distinguish them from the dwarves very very easily, and again fricking hilarious. Amazing work!

My favorite random gen gnome 
Gnome Boy

**Review by Chipawa12 — COMMENTED** [review]
Gnome models are working looking really good!  The little bit of wackiness fits perfectly.

**Pull request merged**